src/components/@shared/FormInput/InputElement.tsx

```diff
@@ -87,6 +87,23 @@ export default function InputElement({
         />
       )
     case 'radio':
+      return (
```

Comment:
do we really need to introduce extra component for this as using same component for radio & checkbox worked out so far, In any case, if new markup is needed then you need to consider all places where radios are already used, right now e.g. the radio labels in the publish form don't come through correctly anymore with this change:

<img width="976" alt="Screen Shot 2022-02-09 at 13 02 33" src="https://user-images.githubusercontent.com/90316/153206878-4453b534-43c6-4203-a2ae-09fc1179d680.png">

Reply:
you're completely right, sorry about that. This has been removed. Now this input has been corrected.